aten/src/ATen/cpu/vec256/vec256_base.h

```diff
@@ -120,8 +120,7 @@ struct Vec256 {
   }
   template<typename... Args,
            typename = std::enable_if_t<(sizeof...(Args) == size())>>
-  Vec256(Args... vals) {
-    values = { vals... };
+  Vec256(Args... vals) : values{vals...}{
```

Comment:
Revert unrelated change?

Reply:
Had to make this change because constructing values as before was not working. Mainly values being an array can only be initialized by `values{...}` and not by assignment, apparently. For some reason it worked before since there were no calls to this constructor in the codepath, but due to my change here, https://github.com/pytorch/pytorch/pull/39341/files#diff-a1ca038eb7df0b763436f68ddd635e3fR1101, now there is.

Reply:
Oh, right.  I thought this was just an auto-format.  Didn't look closely enough. :)